aedttest/simulation_data.py

```diff
@@ -193,7 +193,12 @@ def extract_data(desktop, project_dir, project_name, design_names):
             continue
 
         sweeps = app.existing_analysis_sweeps
-        setup_dict = dict(zip(setups_names, sweeps))
+        setup_dict = {}
+        for setups in setups_names:
+            for s in sweeps:
+                if setups in s:
+                    setup_dict[setups] = s
```

Comment:
```suggestion
            for sweep in sweeps:
                if setups in sweep:
                    setup_dict[setups] = sweep
```

I think this will be more explicit compared to `s`